kcp/setup-kcp-cps.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CPS_KUBECONFIG="${CPS_KUBECONFIG:-$(realpath kcp/cps-kubeconfig)}"
+WORKLOAD_KUBECONFIG="${WORKLOAD_KUBECONFIG:-$HOME/.kube/config}"
+WORKSPACE="gitops-service"
```

Comment:
Thanks for catching this bug. It will wreak havoc if multiple people run e2e tests against the same workspace. I'll append a $RANDOM suffix to make them unique